src/test/scala/com/cognite/sdk/scala/common/OAuth2SessionTest.scala

```diff
@@ -67,7 +64,7 @@ class OAuth2SessionTest extends AnyFlatSpec with Matchers with OptionValues {
         }
 
     val io: IO[Unit] = for {
-      authProvider <- OAuth2.SessionProvider[IO](session, refreshSecondsBeforeTTL = 1)
+      authProvider <- OAuth2.SessionProvider[IO](session, refreshSecondsBeforeTTL = 1, MockK8sServiceTokenProvider())
```

Comment:
This would become `IO("kubernetesServiceToken")` (or `IO.pure` if you like).

Reply:
done